CMakeLists.txt

```diff
@@ -47,16 +47,16 @@ project(qlipper)
 
 set(QLIPPER_VERSION 5.1.1)
 
-option(ENABLE_NETWORK_CLIPBOARD_SHARING "Enables experimental network support for clipboard sharing" OFF)
+option(ENABLE_NETWORK_CLIPBOARD_SHARING "Enables experimental network support for clipboard sharing" ON)
 option(ENABLE_LXQT_AUTOSTART "Enables autostart for LXQt" OFF)
 option(USE_SYSTEM_QXT "Use system Qxt Library for global shortcuts; off for Qt5" OFF)
 
 
 LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-
 find_package(Qt5Widgets "5.6" REQUIRED)
 find_package(Qt5LinguistTools REQUIRED)
+find_package(Qt5Network       REQUIRED)
```

Comment:
This is only needed when the `ENABLE_NETWORK_CLIPBOARD_SHARING` is enabled...not?